src/assets/.gitignore

```diff
@@ -0,0 +1 @@
+appregistry.json
```

Comment:
Ah I see. I was recommending just listing `src/assets/appregistry.json` in the top-level .gitignore file, but this is fine too.